_includes/partners/checklist.html

```diff
@@ -7,6 +7,6 @@
         <li>{{ item.text }}</li>
       {% endfor %}
     </ul>
-    <a class="usa-nav__link caret" href="{{ site.baseurl }}/partners/contact/">Contact us to learn more</a>
+    <a class="usa-nav__link caret" href="{{ site.baseurl }}/partners/business_inquiries/">Contact us to learn more</a>
```

Comment:
Since we use dash delimiters for other paths†, I think we should do the same with this page.

```suggestion
    <a class="usa-nav__link caret" href="{{ site.baseurl }}/partners/business-inquiries/">Contact us to learn more</a>
```

† Example: https://login.gov/help/specific-agencies/trusted-traveler-programs/

Reply:
yes! thank you. updated that here: https://github.com/18F/identity-site/pull/969/commits/d476c090ad86cb7558c28a864fd3b054dd01ecd9